networking/p2p/build.gradle

```diff
@@ -1,13 +1,15 @@
 dependencies {
-  implementation project(':pow')
-  implementation project(':util')
-  implementation project(':storage')
-  implementation project(':ethereum:datastructures')
-  implementation project(':ethereum:statetransition')
   implementation project(':data')
   implementation project(':data:metrics')
+  implementation project(':ethereum:datastructures')
+  implementation project(':ethereum:statetransition')
+  implementation project(':logging')
+  implementation project(':pow')
   implementation project(':services:serviceutils')
+  implementation project(':storage')
+  implementation project(':util')
   implementation project(':validator:coordinator')
+
```

Comment:
For future PRs, it would be helpful to do re-orderings in a separate PR.  If it bothers you feel free to put up a PR that just reorders them all. :)

Reply:
will do 👍